tests/unit/data/rights.test.js

```diff
@@ -8,4 +18,42 @@ describe('checking the format of right.js file', () => {
 
     expect(noDuplicatedRights).toBeTruthy();
   });
+
+  it('a role must have the permissions associated with the role below it (client side)', async () => {
+    const auxiliaryWithoutCompanyPermissions = rights
+      .filter(right => right.rolesConcerned.includes(AUXILIARY_WITHOUT_COMPANY)).map(right => right.permission);
+    const auxiliaryPermissions = rights
+      .filter(right => right.rolesConcerned.includes(AUXILIARY)).map(right => right.permission);
+    const planningReferentPermissions = rights
+      .filter(right => right.rolesConcerned.includes(PLANNING_REFERENT)).map(right => right.permission);
+    const coachPermissions = rights
+      .filter(right => right.rolesConcerned.includes(COACH)).map(right => right.permission);
+    const clientAdminPermissions = rights
+      .filter(right => right.rolesConcerned.includes(CLIENT_ADMIN)).map(right => right.permission);
```

Comment:
tu pourrais faire un it par inclusion ce serait plus clair je pense
et idem pour vendeur

Reply:
done